.github/workflows/amd-offline-tests.yml

```diff
@@ -33,8 +33,8 @@ jobs:
           apt remove -y cmake
           apt update
           apt install -y git
-          python3 -m pip install --upgrade pre-commit
-          python3 -m pre_commit run --all-files
+          # python3 -m pip install --upgrade pre-commit
```

Comment:
Do we expect to re-enable this at some point? I think it may be better to just remove them.

Reply:
n/m, its removed later